.github/workflows/go-tests.yaml

```
@@ -32,9 +32,13 @@ jobs:
                   restore-keys: |
                       ${{ runner.os }}-go-lint-
             - name: Install golangci-lint
-              run: curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b "$(go env GOPATH)/bin" v1.61.0
+              run: curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.61.0
```

actionlint

shellcheck reported issue in this script: SC2046:warning:1:101: Quote this to prevent word splitting

(shellcheck)